odoo/addons/base/models/ir_embedded_actions.py

```diff
@@ -51,6 +51,13 @@ def create(self, vals_list):
         for vals in vals_list:
             if "name" not in vals:
                 vals["name"] = self.env["ir.actions.actions"].browse(vals["action_id"]).name
+            if "python_method" in vals and "action_id" in vals:
+                python_method = vals.get("python_method")
+                if python_method:
```

Comment:
why create a new variable :shrug: 
```suggestion
                if vals.get("python_method"):
```

Reply:
Oops, indeed